### PR TITLE
LPD-26937 Migrate DataSetViewFields poshi cases related to "Cancel" to playwright

### PR DESCRIPTION
**Pull request by dsanz**
In this PR we are migrating many tests related to cancel field selection, edition and deletion to playwright

Sending after checking new tests are green in CI

Reference: https://liferay.atlassian.net/browse/LPD-26936

### CODE REVIEW

**Comment by dsanz** [comment]
ci:test:frontend

File: modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts [hunk 1]
```diff
@@ -459,6 +459,149 @@ test.describe('Visualization Modes in Data Set Manager', () => {
...
+			await visualizationModesPage.selectField({
+				fieldName: SAMPLE_SCALAR_FIELD,
+			});
+
+			await saveFromModal({
+				page,
+			});
+		});
+
+		await test.step('Unselect selected field. Aelect another. Cancel @LPS-185230', async () => {
```
Comment (juanjofgliferay) [inline modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts:507]: Typo _"Aelect another"_

**Review by juanjofgliferay — APPROVED** [review]
LGTM

**Comment by dsanz** [comment]
ci:test:sf

**Comment by dsanz** [comment]
ci:test:sf

**Comment by dsanz** [comment]
ci:test:frontend

**Review by markocikos — APPROVED** [review]
@dsanz LGTM! :sparkles:

**Pull request closed without merging**